package.json

```diff
@@ -3,8 +3,8 @@
   "description": "MetaMask documentation.",
   "scripts": {
     "dev": "yarn start",
-    "start": "vuepress dev docs",
-    "build": "vuepress build docs",
+    "start": "node build.js dev",
+    "build": "node build.js build",
```

Comment:
`vuepress` has a lot more cli niceties than we're exposing with our wrapper script, e.g. `yarn build --help`, and having to maintain the wrapper script is less than ideal. We could simplify this by preloading `dotenv` with the [`-r`](https://nodejs.org/api/cli.html#cli_r_require_module) flag:<sup>[\[1\]][1]</sup>

  [1]:https://www.npmjs.com/package/dotenv#preload

```suggestion
    "start": "node -r dotenv/config node_modules/.bin/vuepress dev docs",
    "build": "node -r dotenv/config node_modules/.bin/vuepress build docs",
```

With this we could avoid neeing the extra `build.js` file.

Reply:
Done in 2d8c946, except the `docs` source directory argument, as that's now in the VuePress config.

Reply:
Nope, the `docs` arg was actually necessary now 🤔 

Whatever, added back `docs` and removed the `sourceDir` config property: 80b9815